src/bulge-pinch/BulgePinchFilter.ts

```diff
@@ -7,6 +8,15 @@ import type { FilterSystem, PointData, RenderSurface, Texture } from 'pixi.js';
 
 // This WebGPU filter has been ported from the WebGL renderer that was originally created by Julien CLEREL (@JuloxRox)
 
+type DeprecatedPointLike = PointData | number[];
+
+interface DeprecatedBulgePinchFilterOptions
+{
+    center: DeprecatedPointLike;
+    radius: number;
+    strength: number;
+}
+
```

Comment:
Again, I'd argue that we could support `number[]` publicly, but simply transform it into a PointData in the center setter. Then you don't need all this deprecation code. Just because unforms don't support `number[]` doesn't mean it's not a valid API choice.